ObsUpdateMsg obs_update
---
bool success
